packages/db/src/core/integration/index.ts

```diff
@@ -120,6 +127,11 @@ function astroDBIntegration(): AstroIntegration {
 	};
 }
 
+function databaseFileEnvDefined() {
+	const env = loadEnv('', process.cwd());
+	return env.DATABASE_FILE != null || process.env.DATABASE_FILE != null;
```

Comment:
nit: Any reason to use `DATABASE_FILE` instead of `ASTRO_DATABASE_FILE`?

I'm also pretty sure `loadEnv` covers the `process.env` check for us, but would need to test

Reply:
It doesn't cover it, the test I have needed the fallback.

No particular reason for the name, happy to change it to `ASTRO_DATABASE_FILE`.